daemon/execdriver/lxc/driver.go

```diff
@@ -214,10 +256,12 @@ func (d *driver) Run(c *execdriver.Command, pipes *execdriver.Pipes, startCallba
 	c.ProcessConfig.Args = append([]string{name}, arg...)
 
 	if err := createDeviceNodes(c.Rootfs, c.AutoCreatedDevices); err != nil {
+		killNetNsProc(proc)
```

Comment:
Because this function does not return until the container has exited. I don't want to keep that netns proc until that. I just want to kill it once we positively know that the lxc has started the container.